src/graph_notebook/magics/graph_magic.py

```diff
@@ -959,67 +987,76 @@ def on_button_clicked(b):
                 queue_hbox.close()
                 dep_hbox.close()
                 ids_hbox.close()
+                nopoll_hbox.close()
                 button.close()
                 output.close()
 
-
                 if 'status' not in load_result or load_result['status'] != '200 OK':
                     with output:
                         print('Something went wrong.')
                         print(load_result)
                         logger.error(load_result)
                     return
 
-                load_id_label = widgets.Label(f'Load ID: {load_result["payload"]["loadId"]}')
-                poll_interval = 5
-                interval_output = widgets.Output()
-                job_status_output = widgets.Output()
-
-                load_id_hbox = widgets.HBox([load_id_label])
-                status_hbox = widgets.HBox([interval_output])
-                vbox = widgets.VBox([load_id_hbox, status_hbox, job_status_output])
-                display(vbox)
-
-                last_poll_time = time.time()
-                while True:
-                    time_elapsed = int(time.time() - last_poll_time)
-                    time_remaining = poll_interval - time_elapsed
-                    interval_output.clear_output()
-                    if time_elapsed > poll_interval:
-                        with interval_output:
-                            print('checking status...')
-                        job_status_output.clear_output()
-                        with job_status_output:
-                            display_html(HTML(loading_wheel_html))
-                        try:
-                            load_status_res = self.client.load_status(load_result['payload']['loadId'])
-                            load_status_res.raise_for_status()
-                            interval_check_response = load_status_res.json()
-                        except Exception as e:
-                            logger.error(e)
+                if nopoll.value == 'TRUE':
```

Comment:
This logic is a little hard to read as its sort of a double negative.  Why not set poll=true if you want to poll and poll=false if you do not.

Reply:
The current check originated from a suggestion that we keep the status polling as the default behavior for bulk load, and add an arg-less `--nopoll` option in case users wanted to disable it. 

This works very well for users that want to specify the arguments directly when executing `%load`, as the usage is clear based on the argument name, and having a predefined argument value provides simplicity. 

However, `--nopoll` does appear to be less beneficial to users that want to use the `%load` widget interface to provide argument values. The possible double negative displayed in the option `No Load Status Polling` -> `[TRUE,FALSE]` could potentially cause confusion. To resolve this, we could change the parameter to be `--poll` instead, and take in a boolean value, with the default remaining as `true`. I do worry that this may cause confusion due to inconsistency with all of the other boolean arguments for `%load`, as `--poll` will have no effect on the loader behavior is specified without an argument value.

Reply:
Done, updated to use `poll_status.value == 'FALSE`